src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -360,10 +362,17 @@ protected final void verify(Checker checker,
             message.append("differing violations: ").append(differingViolations);
         }
 
-        assertTrue(message.toString(),
-                missingViolations.isEmpty()
-                        && unexpectedViolations.isEmpty()
-                        && differingViolations.isEmpty());
+        assertWithMessage(message.toString())
+                .that(missingViolations)
+                .isEmpty();
+
+        assertWithMessage(message.toString())
+                .that(unexpectedViolations)
+                .isEmpty();
+
+        assertWithMessage(message.toString())
```

Comment:
`message.toString()` please extract variable instead of repeating

Reply:
done